src/dsp_tools/commands/validate_data/sparql/cardinality_shacl.py

```diff
@@ -75,20 +70,16 @@ def _construct_1_cardinality(onto_graph: Graph) -> Graph:
           sh:severity sh:Violation ;
           sh:message "1" ;
       ] .
-    
     } WHERE {
-    
       ?class a owl:Class ;
           knora-api:isResourceClass true ;
           knora-api:canBeInstantiated true ;
           rdfs:subClassOf ?restriction .
       ?restriction a owl:Restriction ;
           owl:onProperty ?propRestriction ;
+          salsah-gui:guiOrder ?order ;
```

Comment:
I already created a new ticket to revert it once we added the file values:
https://linear.app/dasch/issue/DEV-4281/change-exclusion-of-properties-from-gui-order-to-editable